ipaserver/install/bindinstance.py

```diff
@@ -775,6 +776,8 @@ def __setup_sub_dict(self):
             NAMED_PID=paths.NAMED_PID,
             NAMED_VAR_DIR=paths.NAMED_VAR_DIR,
             BIND_LDAP_SO=paths.BIND_LDAP_SO,
+            DATA_DIR='' if ipaplatform.NAME == 'debian' else 'data/',
+            DISABLE_DEBIAN='//' if ipaplatform.NAME == 'debian' else '',
             )
```

Comment:
I'm fine with having a separate conffile for bind, just need to first create one that fits Debian better